spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -129,6 +129,18 @@ public StructLikeSet rowSet(String name, Table table, String... columns) {
   public void testEqualityDeleteWithFilter() throws IOException {
     String tableName = "test_with_filter";
     Table table = createTable(tableName, SCHEMA, SPEC);
+
```

Comment:
Why not just use the created `test` table ( in `writeTestDataFile` annotated with a `Before` )  ? then we don't have to write those rows again for this newly created `test_with_filter` table.

Reply:
+1 to what @openinx suggests. Then we also don't need to reason about dropping the table.